app/src/main/AndroidManifest.xml

```diff
@@ -76,10 +80,12 @@
                 <data android:scheme="@string/fb_login_protocol_scheme" />
             </intent-filter>
         </activity>
-        <activity android:name=".Capture"
+        <activity
+            android:name=".Capture"
```

Comment:
Needs to be `CaptureActivity`, not `Capture`.

Reply:
Thanks! Deleted the class